.circleci/config.yml

```diff
@@ -246,32 +204,31 @@ jobs:
       - restore_cache:
           name: Restoring GOCACHE
           keys:
-            - influxdb-gocache-{{ .Branch }}-{{ .Revision }} # Matches when retrying a single run.
-            - influxdb-gocache-{{ .Branch }}- # Matches a new commit on an existing branch.
-            - influxdb-gocache- # Matches a new branch.
+            - influxdb-gotest-{{ .Branch }}-{{ .Revision }} # Matches when retrying a single run.
+            - influxdb-gotest-{{ .Branch }}- # Matches a new commit on an existing branch.
+            - influxdb-gotest- # Matches a new branch.
       # Populate GOPATH/pkg.
       - restore_cache:
           name: Restoring GOPATH/pkg/mod
           keys:
-            - influxdb-gomod-{{ checksum "go.sum" }} # Matches based on go.sum checksum.
+            - influxdb-gotest-gomod-{{ checksum "go.sum" }} # Retrying a single run.
+            - influxdb-gomod-{{ checksum "go.sum" }} # Fresh test-run following the build job.
       - run: mkdir -p $TEST_RESULTS
-      - run: make test-go # This uses the test cache so it may succeed or fail quickly.
       - run:
-          name: parallel go race tests
+          name: run parallel race tests
           command: |
-            set +e
+            GO_TEST_CMD="gotestsum --format standard-quiet --junitfile /tmp/test-results/gotestsum.xml -- -p=4"
             TESTFILES=$(go list ./... | circleci tests split --split-by=timings)
-            echo $TESTFILES
```

Comment:
The `make` command below should echo the commands it executes, including the value of `TESTFILES`